### PR TITLE
Upgrade gradle to version 7.5+

### PR DESCRIPTION
**Pull request by thiartj**
Due to interface changes between Gradle versions 7.4 and 7.5, acquiring a resource file lock needs a different method call.

Resolves issue #50 .

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -19,7 +19,7 @@ plugins {
     // project plugins
     `java-gradle-plugin`
     groovy
-    kotlin("jvm") version "1.4.20"
+    kotlin("jvm") version "1.6.21"
```
Comment (thetric) [inline build.gradle.kts:22]: You might update to Kotlin 1.7

**Comment by marcel-becker** [comment]
Any progress on this MR? I am waiting on this to upgrade my project's gradle version.

File: build.gradle.kts (hunk 1 above)
Comment (thiartj) [reply build.gradle.kts:22]: In theory, yes, but I am not really familiar and Kotlin 1.6+ was simply the minimal required version.

**Comment by thiartj** [comment]
Guess this PR is no longer necessary, as the master branch now includes a Gradle 7.5.1 update.

**Pull request closed without merging**